crates/std_detect/src/detect/arch/riscv.rs

```diff
@@ -157,7 +146,7 @@ features! {
     @FEATURE: #[unstable(feature = "stdarch_riscv_feature_detection", issue = "111192")] zam: "zam";
     /// "Zam" Standard Extension for Misaligned Atomics
 
-    @FEATURE: #[unstable(feature = "stdarch_riscv_feature_detection", issue = "111192")] s: "s";
+    @FEATURE: #[stable(feature = "riscv_ratified", since = "1.76.0")] s: "s";
```

Comment:
None of our feature detection (via hwcaps/cpuinfo) actually enables this feature, so it doesn't make sense to stabilize it.

Reply:
Since the linux kernel does now detect them, should be add them to the linux hwcaps?

<https://github.com/torvalds/linux/blob/master/arch/riscv/include/asm/hwcap.h>

Reply:
For now I would prefer sticking to the same set of features that was stabilized in the compiler itself.